src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableSortedDictionary.cs

```diff
@@ -164,14 +164,14 @@ public static class ImmutableSortedDictionary
         /// <summary>
         /// Returns an immutable copy of the current contents of the builder's collection.
         /// </summary>
-        /// <param name="builder">The builder to create the immutable map from.</param>
+        /// <param name="source">The builder to create the immutable map from.</param>
         /// <returns>An immutable map.</returns>
         [Pure]
-        public static ImmutableSortedDictionary<TKey, TValue> ToImmutableSortedDictionary<TKey, TValue>(this ImmutableSortedDictionary<TKey, TValue>.Builder builder)
+        public static ImmutableSortedDictionary<TKey, TValue> ToImmutableSortedDictionary<TKey, TValue>(this ImmutableSortedDictionary<TKey, TValue>.Builder source)
```

Comment:
This would be an API breaking change. It's likely not a biggie, but given that immutable collections today is using both `builder` and `source` across all the APIs I'd at least make immutable consistent to either use `source` everywhere or `builder`. But just changing one doesn't seem worth it IMHO.

Reply:
This one hasn't ship though. It is new to 3.0.

Reply:
If we're OK with breaking, I think source makes more sense, but I'll leave that call to you.

Reply:
We're OK with breaking code written against earlier 3.0 previews. However we are not OK with breaking code written against Preview 7. So any change here must go in now, before 6/26.

Reply:
Yes, what I meant was changing the APIs that shipped on previous versions (2.0, 2.1, 1.1) -- which is what I was asking, because @terrajobst said that he wanted to be consistent. However I think we should make at least this APIs correct for 3.0

Reply:
From https://github.com/dotnet/corefx/pull/35474#discussion_r258767519

From @ahsonkhan:
> breaking change (src/ref discrepancy): `source` -> `builder` - what is the right parameter name?

From @bartonjs:
> https://github.com/dotnet/corefx/issues/21055 called it `source` when it was proposed, so that seems to be the right answer.

From @ahsonkhan:
> Except every where else `System.Collections.Immutable.ImmutableArray<T>.Builder` arguments are called `builder` :(

Consistency with the rest of the existing APIs might be better. I don't think we use `source` anywhere else. For all other `.Builder` parameters, the name is `builder`.

This seems like the only outlier:
https://github.com/dotnet/corefx/blob/6cb854630083aac733d5b8001b266b872b56748e/src/System.Collections.Immutable/ref/System.Collections.Immutable.cs#L756

Reply:
Makes sense. I'll change the ref to match instead. Thanks for the thread, @ahsonkhan

Reply:
Fixed.